src/helpers/telegram/commands/backup.js

```diff
@@ -10,15 +10,15 @@ module.exports = (ctx) => {
 	const args = command.splitArgs
 
 	let target = { id: user.id.toString(), name: user.first_name }
-	if (!_.includes(controller.config.telegram.admins, user.id.toString()) && ctx.update.message.chat.type === 'group') {
+	if (!_.includes(controller.config.telegram.admins, user.id.toString()) && ctx.update.message.chat.type === 'group' || ctx.update.message.chat.type === 'supergroup') {
```

Comment:
You're breaking admin check 
this translates to 

admin ok + type group 
OR 
anyone in super group